tools/integration_tests/implicitdir/implicitdir_test.go

```diff
@@ -130,9 +155,30 @@ func TestMain(m *testing.M) {
 	}
 
 	log.Printf("Test log: %s\n", logFile)
+
+	// tmp dir setup
+	tmpDir, err := ioutil.TempDir(mntDir, "tmpDir")
+	if err != nil {
+		printAndExit(fmt.Sprintf("Mkdir at %q: %v", mntDir, err))
+	}
+
+	// tmp file setup
+	tmpFile, err = ioutil.TempFile(tmpDir, "tmpFile")
```

Comment:
pls use WriteFile(pathName) method and avoid all this code.

Reply:
Done